### ISSUE TITLE
Error when I run the npm start command.

### ISSUE THREAD

**Issue by freddy231284** [qdouble/angular-webpack2-starter#117]
Error when I run the npm start command.
  I get the following error:
That solution can give you is That please?

![image](https://cloud.githubusercontent.com/assets/22718916/19216721/f2725ad8-8d8d-11e6-99a2-53536127c9be.png)

**Comment by qdouble** [comment]
@freddy231284 which branch are you using? is this after a fresh clone and install?

**Comment by freddy231284** [comment]
hi,  I am using this repository, https://github.com/qdouble/angular2webpack2-starter/tree/no-universal-support,
and if after clone the repository, I update the repository with
git checkout non-universal-support, and then perform the installation with the command npm install

**Comment by qdouble** [comment]
@freddy231284 I did a fresh install on Windows 10 using VM and was unable to reproduce the issue, but after google searching it appears that it's a problem with your node-sass installation (either do to permission or faulty install).

You want to see if this link helps: https://github.com/sass/node-sass/issues/1579
Or just google search for 'node sass no such file or directory' and see if any of the search results help.

Closing as it doesn't appear to be a problem with this repo itself.
